src/context/index.tsx

```diff
@@ -21,10 +22,10 @@ const LDContext = createContext<Context>(null)
 
 const LDProvider: FC<Props> = ({ initialLDData, clientId, children }) => {
   // persists the data initialized server-side on the client
-  const ldData = useMemo(() => initialLDData, [])
+  const ldData = useMemo(() => initialLDData, [initialLDData])
```

Comment:
have you tested that this does only run once? I'm not sure if the reference is stable

Reply:
I've tested it and it runs only once per page. That's the correct behavior